intl-memoizer/src/lib.rs

```diff
@@ -407,4 +408,46 @@ mod tests {
             assert_eq!(result, Ok(PluralCategory::OTHER));
         }
     }
+
+    #[test]
+    fn test_concurrent() {
+        let lang: LanguageIdentifier = "en".parse().unwrap();
+        let memoizer = Arc::new(concurrent::IntlLangMemoizer::new(lang));
+        let mut threads = vec![];
+
+        // Spawn four threads that all use the PluralRules.
+        for _ in 0..4 {
+            let memoizer = Arc::clone(&memoizer);
+            threads.push(thread::spawn(move || {
+                memoizer
+                    .with_try_get::<PluralRules, _, _>((PluralRuleType::CARDINAL,), |cb| {
+                        cb.0.select(5)
+                    })
+                    .expect("Failed to get a PluralRules result.")
+            }));
+        }
+
+        for thread in threads.drain(..) {
+            let result = thread.join().expect("Failed to join thread.");
+            assert_eq!(result, Ok(PluralCategory::OTHER));
+        }
+    }
+
+    #[test]
+    fn test_debug_concurrent() {
+        let memoizer = concurrent::IntlLangMemoizer::new("en".parse().unwrap());
+        assert!(
+            format!("{:?}", memoizer).starts_with("IntlLangMemoizer"),
+            "The memoizer debug prints."
+        );
+    }
+
+    #[test]
+    fn test_debug_single_thread() {
+        let memoizer = IntlLangMemoizer::new("en".parse().unwrap());
+        assert!(
+            format!("{:?}", memoizer).starts_with("IntlLangMemoizer"),
+            "The memoizer debug prints."
+        );
+    }
```

Comment:
> thought

I'm a bit unclear on what this is testing that is of use. Any object that derives `Debug` will print _something_ that contains the object's type name.

Reply:
I was dogmatically testing anything that was uncovered, which the Debug trait was being listed as uncovered. Maybe it's not the best use of a test.

Reply:
Hmm. My preference would be if there was a way to ignore `Debug` trait from the coverage report. I don't know if we have that ability, but I would like that more than having tests that don't really test anything substantial just for the sake of having coverage here. It's not the end of the world to keep them, but I don't think we gain anything from this other than a higher number on the report.

Reply:
Weirdly enough, the coverage isn't even showing up as missing there now.